apps/omg/test/omg/state/core_test.exs

```diff
@@ -897,6 +898,148 @@ defmodule OMG.State.CoreTest do
            )
   end
 
+  describe "Automatic fees claiming" do
+    setup do
+      fee_claimer = OMG.TestHelper.generate_entity()
+
+      {:ok, child_block_interval} = OMG.Eth.RootChain.get_child_block_interval()
+      {:ok, state} = Core.extract_initial_state(0, child_block_interval, fee_claimer.addr)
+
+      alice = OMG.TestHelper.generate_entity()
+      fees = %{@eth => %{amount: 2}}
+
+      # Transaction requires fee of 2wei ETH but Alice actually is paying 3wei
```

Comment:
Note that this should fail once @T-Dnzt prevent over-payment

Reply:
I guess there would be still some margin :) We'll see :D